yt_dlp/extractor/puhutv.py

```diff
@@ -191,27 +192,29 @@ def _extract_entries(self, seasons):
             if not season_id:
                 continue
             page = 1
+            per = 100
             has_more = True
             while has_more is True:
                 season = self._download_json(
-                    f'https://galadriel.puhutv.com/seasons/{season_id}',
-                    season_id, f'Downloading page {page}', query={
+                    f'https://appservice.puhutv.com/api/seasons/{season_id}/episodes?v=2',
+                    season_id, f'Downloading episode {(page - 1) * per}-{page * per} metadata', query={
                         'page': page,
```

Comment:
I'd prefer to keep the note as it was before. If we say we're downloading `episode 400-500` but there are only `456` episodes, this would be confusing for the user
```suggestion
                    f'https://appservice.puhutv.com/api/seasons/{season_id}/episodes?v=2',
                    season_id, f'Downloading page {page}', query={
                        'page': page,
```